mlir/lib/Dialect/Linalg/TransformOps/LinalgTransformOps.cpp

```diff
@@ -1180,6 +1180,15 @@ transform::MatchOp::apply(transform::TransformRewriter &rewriter,
         return;
     }
 
+    if (getFilterOperandType().has_value()) {
+      Type t = getFilterOperandType().value();
+      for (auto type : op->getOperandTypes()) {
```

Comment:
nit: Can use use `!llvm::all_of`. Also, drop trivial braces.

Reply:
Solved, although I personally feel that with `llvm::all_of` it is less readable